client/src/document/ingredients/polyfills-subsection.tsx

```diff
@@ -0,0 +1,66 @@
+import { DisplayH3 } from "./utils";
+
+const ALLOWED_POLYFILL_SERVICE = [
+  {
+    baseURL: "https://github.com/zloirock/core-js",
+    serviceURL: "",
+    serviceName: "core-js",
+  },
+  {
+    baseURL: "https://github.com/Financial-Times/polyfill-service",
+    serviceURL: "http://polyfill.io",
+    serviceName: "polyfill.io",
+  },
+  {
+    baseURL: "https://formatjs.io/docs/polyfills",
+    serviceURL: "https://formatjs.io/",
+    serviceName: "formatjs.io",
+  },
+  {
+    baseURL: "https://github.com/tc39/",
+    serviceURL: "https://github.com/tc39/",
+    serviceName: "TC39",
+  },
+  {
+    baseURL: "https://github.com/w3c",
+    serviceURL: "https://github.com/w3c",
+    serviceName: "W3C",
+  },
+];
+
+function polyfillService(url) {
```

Comment:
Could be more descriptive:

```suggestion
function getPolyfillServiceByUrl(url) {
```